### PR TITLE
Add _index.md for multicluster examples

### PR DESCRIPTION
**Pull request by sdake**

### CODE REVIEW

File: content/docs/examples/multicluster/_index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: Multicluster Enablement
```
Comment (geeknoid) [inline content/docs/examples/multicluster/_index.md]: How about "Enabling multiclusters"?

File: content/docs/examples/multicluster/_index.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+description: A variety of fully working multicluster examples for Istio that you can experiment with.
+weight: 100
+type: section-index
+page_icon: /img/examples.svg
+---
```
Comment (geeknoid) [inline content/docs/examples/multicluster/_index.md]: Add
keywords: [multicluster]

File: content/docs/examples/multicluster/_index.md (hunk 1 above)
Comment (sdake) [reply content/docs/examples/multicluster/_index.md]: done

File: content/docs/examples/multicluster/_index.md (hunk 2 above)
Comment (sdake) [reply content/docs/examples/multicluster/_index.md]: done

**Pull request merged**